client/src/common/components/ModalForm.scss

```diff
@@ -9,6 +9,7 @@
   z-index: 10;
   min-width: 320px;
   max-width: 600px;
+  border-radius: 3px;
```

Comment:
In our landing page there is no border-radius, so I wonder if should use it here, maybe like in buttons all corners should be right angles...

Reply:
I have tried with no radius - it looks ugly in my opinion. I'm looking forward to get some feedback from Lukasz